### PR TITLE
use ng-submit

### LINKED ISSUE #382: Pressing enter in new calendar input doesn’t create calendar
You need to manually click the button, which is quite annoying. Pressing enter after having put in a calendar name should also work.

(Observed when I watched @nickvergessen use the the Calendar app ;)

### PR DESCRIPTION
**Pull request by georgehrke**
fixes https://github.com/owncloud/calendar/issues/382

please review @raghunayyar @jancborchardt

### CODE REVIEW

**Comment by BernhardPosselt** [comment]
Btw this should work out of the box when using forms, also check th ngSubmit directive: https://docs.angularjs.org/api/ng/directive/ngSubmit

**Comment by BernhardPosselt** [comment]
Check this: https://github.com/owncloud/news/blob/master/templates/part.navigation.addfeed.php

TL;DR: bind to ng-submit and not ng-click for the submit button

**Comment by georgehrke** [comment]
@BernhardPosselt Does ng-submit also catch the enter?

**Comment by BernhardPosselt** [comment]
Yes, everything that you'd expect from a form. And it prevents page reload ;)

**Comment by georgehrke** [comment]
okay, will try out, thx :)

**Comment by raghunayyar** [comment]
:+1:
